nginx.conf

```diff
@@ -27,6 +27,7 @@ http {
   server {
     listen <%= ENV["PORT"] %>;
     server_name localhost;
+	include    /usr/local/etc/nginx/redirects.conf;
```

Comment:
as long as the file is in the same directory as `nginx.conf` you just need the file name.

`include redirects.conf`